protocol/Channel.py

```diff
@@ -28,8 +27,8 @@ def broadcast(self, message, ignore=set([])):
 	def channelMessage(self, message):
 		self.broadcast('CHANNELMESSAGE %s %s' % (self.name, message))
 
-	def register(self, client, owner_user_id):
-		self.owner_user_id = owner.db_id
+	def register(self, client, owner_user_id): # fixme: unused?
```

Comment:
when unused: remove it? :)

Reply:
It is unused here - but it is used in the other PR! So better to leave it in here, I think.